scripts/src/middleware/session.ts

```diff
@@ -0,0 +1,34 @@
+import * as session from "koa-session";
+import * as Koa from "koa";
+import Keygrip = require("keygrip");
+
+const DEFAULT_CONFIG: Config = {
+	// /**
```

Comment:
all of the commented out code and comments are needed?
it looks like it's copied from somewhere, maybe just add the url as a comment above?